lib/web_ui/test/engine/semantics/semantics_test.dart

```diff
@@ -2,7 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-// @dart = 2.6
+// @dart = 2.9
```

Comment:
Should this be 2.12 at this point?

Reply:
2.12 only if we migrate the test to null-safety, which this is not doing yet :P